docs/editor/extension-gallery.md

```diff
@@ -129,27 +132,37 @@ To make it easier to automate and configure VS Code, it is possible to list, ins
 Example:
 
 ```
+code --extensions-dir <dir>
```

Comment:
@ramya-rao-a here I didn't want to add just a picture because all of the information would be in the picture, rather than the picture just supporting the text, so we'd have to have the same exact content again in the alt-text

Reply:
agreed